flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -160,31 +160,6 @@
 			.defaultValue("32kb")
 			.withDescription("Size of memory buffers used by the network stack and the memory manager.");
 
-	/**
-	 * Amount of memory to be allocated by the task manager's memory manager. If not
-	 * set, a relative fraction will be allocated, as defined by {@link #MANAGED_MEMORY_FRACTION}.
-	 */
-	@Deprecated
-	public static final ConfigOption<String> LEGACY_MANAGED_MEMORY_SIZE =
-			key("taskmanager.memory.size")
```

Comment:
we still have mentions of raw config name strings like `taskmanager.memory.size` in some places, e.g. in docs. I think it would be good to update those places, also for other removed options.